pkg/comp-functions/functions/vshnnextcloud/deploy.go

```diff
@@ -256,10 +257,20 @@ func newValues(ctx context.Context, svc *runtime.ServiceRuntime, comp *vshnv1.VS
 		}
 	}
 
+	if len(comp.Spec.Parameters.Service.FQDN) == 0 {
```

Comment:
Can we add that into directly in the the webhook?

Reply:
Done

Reply:
> Can we add that into directly in the the webhook?

Already covered here: https://github.com/vshn/appcat/pull/269/files#diff-d41d9990ae83874d678984536634bbcc08146417526791f235c09d1bdc9f0d7eR94

No need for a webhook here...

Reply:
True, I've removed the whole section